example/lib/behaviors/movement_behavior.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flame/extensions.dart';
+import 'package:flame_behaviors/flame_behaviors.dart';
+
+class MovementBehavior extends Behavior {
```

Comment:
Could be provided by the packge itself?

Reply:
Hmm that is a good point, I will think about that over the weekend. On Flame we don't have a velocity field simply because we said "that really depends on the game". But when it comes to behaviors it wouldn't be strange to provide that, as Dart will shake it out of the tree if it isn't used anyway.